src/app/containers/ContactsContainer.js

```diff
@@ -248,11 +248,13 @@ const ContactsContainer = ({ location, history }) => {
         />
     );
 
+    const title = search === '' ? c('Title').t`Contacts` : c('Title').t`Search`;
```

Comment:
I think what #351 wants is:
```
const title = isNarrow ? c('Title').t`Search` : c('Title').t`Contacts`;
```
Actually "Contacts" will never be displayed in the current scheme of things